src/backend/api/meals.js

```diff
@@ -2,7 +2,10 @@ const express = require("express");
 const router = express.Router();
 const knex = require("../database");
 
-router.get("/", async (request, response) => {
+let date = new Date();
```

Comment:
When the value is not meant to be changed later on, it is a good idea to use const instead of let.

Reply:
You are right. Thank you.